### DISCUSSION TITLE
aquaCrop_OSpy notebook for tutorial key error 'Yield (tonne/ha)'

### DISCUSSION THREAD

**Discussion by shaiabir** [aquacropos/aquacrop#175] [Q&A]
in all tutorial notebooks after running the model when using the output pandas data frame it is expected to get a columns named 'Yield (tonne/ha)' for instance in notebook 3:
```python
    yld = out['Yield (tonne/ha)'].mean()

``` 
however I think this is a legacy from previous model version and this should be 'Dry Yield (tonne/ha)'

**Comment by Rimagination** [comment]
yes, I think you are right.
